torch/_inductor/ir.py

```diff
@@ -185,6 +185,7 @@ def _check_tensorbox(nodes: Optional[_NodeOrNodes]) -> None:
                     sympy.logic.boolalg.Boolean,
                     Expr,
                     int,
+                    float,
```

Comment:
If we always spec when we fail to eliminate SymFloat, we shouldn't actually hit this case of the assert, right?